docs/source/API/core/parallel-dispatch/parallel_for.rst

```diff
@@ -81,6 +84,18 @@ More Detailed Examples are provided in the ExecutionPolicy documentation.
         Kokkos::finalize();
     }
 
+.. raw:: html
+
+   <a href="https://godbolt.org/z/q9h339vob" target="_blank">Edit on Compiler Explorer</a>
```

Comment:
`code-block` above + a manually created shortlink to Compiler Explorer:
![image](https://github.com/kokkos/kokkos-core-wiki/assets/4639165/edb4e9a8-627e-4707-af09-acaef3f63c00)